.github/workflows/ci.yml

```diff
@@ -0,0 +1,182 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+    lint-shellcheck:
+        runs-on: ubuntu-latest
+        steps:
+          - uses: actions/checkout@v2
+
+          - name: Runs shell script static analysis
+            run: |
+              sudo apt-get install shellcheck
+              ./run-tests.sh --check-shellscript
+
+    lint-black:
+        runs-on: ubuntu-latest
+        steps:
+          - uses: actions/checkout@v2
+
+          - name: Setup Python
+            uses: actions/setup-python@v2
+            with:
+              python-version: 3.8
+
+          - name: Check Python code formatting
+            run: |
+              pip install black==19.10b0
```

Comment:
Crazy idea: Would it be possible to install `black` from [pytest-reana](https://github.com/reanahub/pytest-reana/blob/master/setup.py#L37)? So we can avoid amending all the GitHub action workflow files in case we decide to switch version.

Worst case scenario we can install `pytest-reana` altogether although it seems overkill

Reply:
For MVP, since we are blocked by the current CI situation, I'd say that anything will do, whatever is easier to implement :wink:  If we decide to say upgrade `black` version in `r-commons`, we ran run an automated multi-`sed` job to upgrade all the workflow definitions. (However, installing `pytest-reana` is naturally also OK, provided it does not slow things down too much or won't bring other f-string or DB circular import issues and friends.)

Reply:
FYI this exact discussion was initially started at https://github.com/reanahub/reana/pull/433#discussion_r512544718.

Reply:
I had missed that, thanks. Good to see we have similar opinions 😀